Software/src/wizard/ZonePlacementPage.cpp

```diff
@@ -162,7 +162,9 @@ bool ZonePlacementPage::checkZoneIssues()
 	QStringList gapStrs;
 	int prevId = -1;
 	QList<int> overlapIds;
-	for (const int id : ids.keys()) {
+	QMultiMap<int, std::nullptr_t>::const_iterator idIt = ids.constBegin();
+	while (idIt != ids.constEnd()) {
+		const int id = idIt.key();
```

Comment:
Is this making anything better?

At least in terms of readability, it looks like a downgrade...
Same for _grabAreas above.

Reply:
avoids allocating keys/values lists just for looping thru